### PR TITLE
14 add clock and window parameters to totp

### PR DESCRIPTION
**Pull request by angelosarto**
I have followed the correct process and managed to submit the code which works and is correct.
If you feel the feature may still not be appropriate (the window) parameter let me know.  I also added new test cases to prove the functionality.

### CODE REVIEW

File: onetimepass/__init__.py [hunk 1]
```diff
@@ -143,6 +144,8 @@ def get_totp(
     :type token_length: int
     :param interval_length: length of TOTP interval (30 seconds by default)
     :type interval_length: int
+    :param clock: clock time in epoch seconds to generate the totp for, default is Now if None
+    :type clock: number
```
Comment (tadeck) [inline onetimepass/__init__.py]: Should be `int` instead of `number`. At least this is my impression.

File: onetimepass/__init__.py [hunk 2]
```diff
@@ -217,6 +222,8 @@ def valid_totp(
         digest_method=hashlib.sha1,
         token_length=6,
         interval_length=30,
+        clock=None,
+        window=0
```
Comment (tadeck) [inline onetimepass/__init__.py]: Please end parameters list with comma - will be easier to detect changes in history.

File: onetimepass/__init__.py [hunk 3]
```diff
@@ -231,6 +238,11 @@ def valid_totp(
     :type token_length: int
     :param interval_length: length of TOTP interval (30 seconds by default)
     :type interval_length: int
+    :param clock: clock time in epoch seconds to generate the totp for, default is Now if None
+    :type clock: number
```
Comment (tadeck) [inline onetimepass/__init__.py]: We also expect `int` here, right?

Comment (angelosarto) [inline onetimepass/__init__.py]: Corrected and created as a new pull request.

**Pull request closed without merging**